### ISSUE TITLE
Failing test(s) due to different resize output on Windows and Linux

### ISSUE THREAD

**Issue by KamilZielinski** [elixir-mogrify/mogrify#90]
I tried to run tests on Windows but for some reason, my image was off by 1px which caused tests to fail.
One of them that I focused on:
```elixir
test ".save in place when file name has spaces" do
    # setup, make a copy
    File.mkdir_p!(@temp_test_directory)
    open(@fixture) |> save(path: @temp_image_with_space)

    # test begins
    image = open(@temp_image_with_space) |> resize("600x600") |> save(in_place: true) |> verbose
    assert %Image{path: @temp_image_with_space, height: 584, width: 600} = image

    File.rm_rf!(@temp_test_directory)
  end
```
The results it gives me are:
- Windows: 600x**585** (45.5kb)
- Linux: 600x**584** (49kb)

It seems like due to different OS (as I tested that on WSL and Windows on the same machine, so architecture shouldn't make a difference here) it gives me different results. Maybe it's related to [that issue](https://stackoverflow.com/a/6381049/3878436) as well, I'm not sure now as I don't know how ImageMagick works underneath.
I think we should find a better way to compare it between these 2 different systems. I'll be trying to solve it but it would be good if you could put some ideas here as well as maybe you are aware of something I could use.
A simple, dirty but working idea for now could be doing asserts depending on the system they run on the same as we did for running mogrify commands on Windows in the past as they work a little bit different. WDYT?
I'll also check other tests if the issue is somehow common so we can solve it globally in the same way.

**Comment by talklittle** [comment]
Interesting issue. Thanks for reporting.

IMO we can assert width == 600 and height in [584, 585], with a comment as to why we are doing that.

**Comment by talklittle** [comment]
Closing this issue. I did not see this issue during testing on various ImageMagick versions 6.9.x and 7.x, Windows and Linux.

It was probably an upstream ImageMagick bug, which is no longer present in current IM versions. It's really hard to imagine when it would ever be correct to resize a 300x292 image to 600x585, when asked to resize up to a 600x600 bounding box. It's not like there's fractional arithmetic involved; it's an exact 2x multiplier.
